source/beamline/BioXAS/BioXASFrontEndShutters.cpp

```diff
@@ -46,7 +45,7 @@ void BioXASFrontEndShutters::setUpstreamPhotonShutter(AMReadOnlyPVControl *newCo
 		upstreamPhotonShutter_ = newControl;
 
 		if (upstreamPhotonShutter_)
-			addShutter(upstreamPhotonShutter_, 1, 4);
+			addShutter(upstreamPhotonShutter_, CLS_SHUTTER_OPEN, CLS_SHUTTER_CLOSED);
```

Comment:
Quick question. Do these `CLS_SHUTTER_OPEN/CLOSED` values represent the PV open/closed, or the pseudomotor CLSShutters open/closed? Remember that those will be different values.